ComponentTextKit/CKTextComponent.mm

```diff
@@ -65,6 +65,7 @@ + (instancetype)newWithTextAttributes:(const CKTextKitAttributes &)attributes
                        viewAttributes:(const CKViewComponentAttributeValueMap &)viewAttributes
                  accessibilityContext:(const CKTextComponentAccessibilityContext &)accessibilityContext
 {
+  CKAssertTrue(!attributes.customLayoutManagerClass || [attributes.customLayoutManagerClass isSubclassOfClass:[NSLayoutManager class]]);
```

Comment:
*Seems* worth having a sanity check assertion for this somewhere, and this seems like as good a place as any. Please let me know about this one.